script/brave_license_helper.py

```diff
@@ -31,10 +32,120 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'vendor', 'brave-ios'),
         os.path.join('brave', 'vendor', 'brave_base'),
 
-        # These have auto-generated license files and
-        # GetThirdPartyDepsFromGNDepsOutput causes strange license errors
-        # unless the this entire directory is excluded.
-        os.path.join('brave', 'third_party', 'rust'),
+        # Metadata files for Rust crates are located in the subfolders of
+        # brave/third_party/rust/<crate_name>/<v>, the crates themselves in
+        # brave/third_party/rust/chromium_crates_io can be skipped.
+        os.path.join('brave', 'third_party', 'rust', 'chromium_crates_io'),
+
+        # Same for upstream
+        os.path.join('third_party', 'rust', 'chromium_crates_io'),
+
+        # Rust code written by Brave.
```

Comment:
```suggestion
        # Rust code written by Brave and under the same license as the browser.
```

Reply:
Added in https://github.com/brave/brave-core/pull/23822/commits/fa3bd5282fcc3aa5afe6272bbbfeff1f75c01b4d